app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig

```diff
@@ -10,14 +10,15 @@
                 <i class="fa fa-navicon fs-16"></i>
             </a>
         </li>
-        {{ render(controller('Mautic\\CoreBundle\\Controller\\DefaultController::notificationsAction')) }}
         {{ render(controller('Mautic\\CoreBundle\\Controller\\DefaultController::globalSearchAction')) }}
     </ul>
     <!--/ end: left nav -->
 
     <!-- start: right nav -->
     <ul class="nav navbar-nav navbar-right">
+        
```

Comment:
```suggestion
        
```

Reply:
Hm, well noted! Thanks

Reply:
_This commit didn't work bc GitHub says no changes were made to the code_